kodi_jellyfin/connect.py

```diff
@@ -3,6 +3,7 @@
 ##################################################################################################
 
 import logging
+import os
```

Comment:
I don't see `os` used anywhere in this file.  Is it needed?

Reply:
Was going to leave the code refactoring to another PR but I can remove it if you think it'd be okay